### PR TITLE
add file Topic Wise Questions with Solutions/Array/Rearrange_the_neag…

### PR DESCRIPTION
**Pull request by soumik2012**
### Have you read the Contributing Guidelines on Pull Requests?

Yes 

### Description

using the single-pass approach 

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.

## Related Issues or Pull Requests

Issue is related #350

### CODE REVIEW

**Comment by soumik2012** [comment]
@sayantani11 please check my PR

File: Topic Wise Questions with Solutions/Array/Rearrange_the_neagtive_positive_element.py [hunk 1]
```diff
@@ -0,0 +1,54 @@
+"""
+Given an array of positive and negative numbers, arrange them in an alternate 
+fashion such that every positive number is followed by negative and vice-versa 
+maintaining the order of appearance. 
+Number of positive and negative numbers need not be equal. 
+If there are more positive numbers they appear at the end of the array.
+If there are more negative numbers, they too appear in the end of the array.
```
Comment (sayantani11) [inline Topic Wise Questions with Solutions/Array/Rearrange_the_neagtive_positive_element.py:7]: It says if the negative numbers is more then they appear in the end , that means positive numbers appear infront

File: Topic Wise Questions with Solutions/Array/Rearrange_the_neagtive_positive_element.py [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+    negi=[]
+    
+    for i in range(len(arr)):
+        if arr[i]>=0:
+            posi.append(arr[i])
+        else:
+            negi.append(arr[i])
+    
+    l=len(arr)
+
```
Comment (sayantani11) [inline Topic Wise Questions with Solutions/Array/Rearrange_the_neagtive_positive_element.py:29]: Shouldn't there also be a count variable that will maintain the count of number of elements in posi array and negi array and then append it.

File: Topic Wise Questions with Solutions/Array/Rearrange_the_neagtive_positive_element.py [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+    arr2=[]
```
Comment (sayantani11) [inline Topic Wise Questions with Solutions/Array/Rearrange_the_neagtive_positive_element.py:30]: If not I guess then you need to make a slight change in the application part

**Review by sayantani11 — CHANGES_REQUESTED** [review]

File: Topic Wise Questions with Solutions/Array/Rearrange_the_neagtive_positive_element.py (hunk 1 above)
Comment (soumik2012) [reply Topic Wise Questions with Solutions/Array/Rearrange_the_neagtive_positive_element.py:7]: yes, they have appeared on the front

File: Topic Wise Questions with Solutions/Array/Rearrange_the_neagtive_positive_element.py (hunk 2 above)
Comment (soumik2012) [reply Topic Wise Questions with Solutions/Array/Rearrange_the_neagtive_positive_element.py:29]: okay

File: Topic Wise Questions with Solutions/Array/Rearrange_the_neagtive_positive_element.py (hunk 3 above)
Comment (soumik2012) [reply Topic Wise Questions with Solutions/Array/Rearrange_the_neagtive_positive_element.py:30]: what I have to change explanation part can you tell me

Comment (soumik2012) [reply Topic Wise Questions with Solutions/Array/Rearrange_the_neagtive_positive_element.py:30]: @sayantani11 which part I have to explain

Comment (soumik2012) [reply Topic Wise Questions with Solutions/Array/Rearrange_the_neagtive_positive_element.py]: @sayantani11  I did explain please check new files

**Review by sayantani11 — APPROVED** [review]

**Comment by soumik2012** [comment]
@rcoder23 please add lable and merge the my PR

**Comment by soumik2012** [comment]
@sayantani11 please add a gssoc'21 and level 3 label also merge my PR

**Comment by soumik2012** [comment]
@sayantani11 Please add the GSSOC label  and level 3 and merge the PR if it's satisfying to you

**Comment by soumik2012** [comment]
@sayantani11 

> @sayantani11 Please add the GSSOC label and level 3 and merge the PR if it's satisfying to you

@sayantani11

**Review by rcoder23 — APPROVED** [review]

**Pull request merged**